pipelines/library/src/common/VersionInfo.groovy

```diff
@@ -225,6 +240,10 @@ class VersionInfo {
         if (major != null) {
             def semver = major + "." + minor + "." + security
 
+            if (fix) {
+                semver += "." + fix
```

Comment:
not sure about this, according to the semver standard you can only have 3 numbers, not sure we can do this and still call it a semver

Reply:
See what you mean, I didn't realize is was a standard! Which means OpenJDK are not semver compliant now...
Shall we just keep semver the same?

Reply:
same as in just major.minor.security